app/views/contact/govuk/anonymous_feedback_thankyou.html.erb

```diff
@@ -1,7 +1,19 @@
-<% title = "Thank you for contacting GOV.UK" %>
-<% content_for :title, title %>
-<p class="govuk-body" 
-   data-module="ga4-auto-tracker"
-   data-ga4-auto="<%= { event_name: "form_complete", type: "contact", text: title, action: "complete", tool_name: "Contact GOV.UK", section: title }.to_json %>">
+<%
+  title = "Thank you for contacting GOV.UK"
+  content_for :title, title
+  data_attributes = {
+    module: "ga4-auto-tracker",
+    ga4_auto: {
+      event_name: "form_complete",
+      type: "contact",
+      text: title,
+      action: "complete",
+      tool_name: "Contact GOV.UK",
+      section: title,
+    }.to_json
+  } if params[:govuk_contact_form]
```

Comment:
Oh right, here. Hmm. I don't know how I feel about accessing params directly in a controller.